experiments/ClimaEarth/user_io/arg_parsing.jl

```diff
@@ -0,0 +1,252 @@
+"""
+    get_coupler_config()
+
+Reads in the configuration file and job ID from the command line.
+A dictionary is constructed from the input configuration file, and this
+is returned along with the job ID.
+
+# Returns
+- `config_dict`: A dictionary mapping configuration keys to the specified settings
+- `job_id`: A unique identifier for the job
+"""
+function get_coupler_config()
+    ## coupler simulation default configuration
+    parsed_args = parse_commandline(argparse_settings())
+
+    ## modify parsed args for fast testing from REPL #hide
+    if isinteractive()
+        parsed_args["config_file"] =
+            isnothing(parsed_args["config_file"]) ? joinpath(pkg_dir, "config/ci_configs/interactive_debug.yml") :
+            parsed_args["config_file"]
+        parsed_args["job_id"] = "interactive_debug"
+    end
```

Comment:
I think we should remove this. We can provide a way `debug_config` function if there's such a need

Reply:
Agreed. We can just add default values for `config_file` and `job_id`, that will get used if we run interactively without specifying alternatives